packages/cli/src/tests/utils/build.js

```diff
@@ -57,7 +57,7 @@ describe('build (runs slowly)', function () {
     smartPaths.filter((p) => p.endsWith('.ts')).length.should.equal(0);
     smartPaths.should.not.containEql('tsconfig.json');
 
-    smartPaths.length.should.be.within(200, 301);
+    smartPaths.length.should.be.within(200, 303);
```

Comment:
[This kept failing ](https://github.com/zapier/zapier-platform/actions/runs/13035582314/job/36365075151?pr=957) consistently after the new schema additions, is this expected?

Reply:
Not sure why, but I think it is fine. It was bumped recently in this [commit](https://github.com/zapier/zapier-platform/commit/37b2b614d43bd88d0d01ba52fd86a5b07d54e92b#diff-0d2de618f30d0923bb5f97731eccdaae00205f82f016b607f5fe9ca46022069c) too.